cloud/scope/machine.go

```diff
@@ -100,20 +106,42 @@ func NewMachineScope(params MachineScopeParams) (*MachineScope, error) {
 		core.SetLoggingLevel(core.LevelDebug)
 	}
 
+	auth, err := authenticator.GetAuthenticator()
+	if err != nil {
+		return nil, fmt.Errorf("error failed to create authenticator: %w", err)
+	}
+
+	// Create Global Tagging client.
+	gtOptions := globaltagging.ServiceOptions{
+		GlobalTaggingV1Options: &globaltaggingv1.GlobalTaggingV1Options{
+			Authenticator: auth,
+		},
+	}
```

Comment:
Is it possible to avoid this here and reuse the auth in globaltagging.NewService(gtOptions) and set it there itself?

Reply:
done

Reply:
where are the supporting changes in globaltagging service function? Not part of this PR?

Reply:
I hope you tested with above change and it is working fine?